portals/publisher/src/main/webapp/source/src/app/components/Apis/Details/Documents/Listing.jsx

```diff
@@ -98,17 +102,38 @@ const styles = theme => ({
         fontWeight: 200,
         marginBottom: 20,
     },
+    subtitle: {
```

Comment:
Generally, you can use the Typography component `variant` prop to set different size text without setting the font sizes manually.